README.md

```diff
@@ -15,68 +15,75 @@ Begin by cloning, and entering this repository on your account on LDG:
 
 > **Note** Ensure that you have added a [github ssh key](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/adding-a-new-ssh-key-to-your-github-account) to your account 
 
-```
+```console
 git clone git@github.com:ml4gw/quickstart.git
 cd quickstart
 ```
 
-Then, simply running `make` at the command line will:
```

Comment:
Maybe it makes sense to make this comment a `**Note**` just so folks know they can run all the subcommands  all at once with a single command if they desire

Reply:
The problem is that they can't. The `bashrc` file needs to be sourced after the initial conda install for poetry and helm to work, and there doesn't seem to be a good way to do that from the Makefile

Reply:
Got it - okay this is fine then